### PR TITLE
Add new model, accton-as5916-54x.

### PR DESCRIPTION
**Pull request by brandonchuang**

### CODE REVIEW

File: packages/platforms/accton/x86-64/x86-64-accton-as5916-54x/onlp/builds/src/module/inc/x86_64_accton_as5916_54x/x86_64_accton_as5916_54x_config.h [hunk 1]
```diff
@@ -0,0 +1,137 @@
+/**************************************************************************//**
+ *
+ * @file
+ * @brief x86_64_accton_as5916_54x Configuration Header
+ *
+ * @addtogroup x86_64_accton_as5916_54x-config
+ * @{
+ *
+ *****************************************************************************/
+#ifndef __x86_64_accton_as5916_54x_CONFIG_H__
```
Comment (jnealtowns) [inline packages/platforms/accton/x86-64/x86-64-accton-as5916-54x/onlp/builds/src/module/inc/x86_64_accton_as5916_54x/x86_64_accton_as5916_54x_config.h]: Uppercase please.

File: packages/platforms/accton/x86-64/x86-64-accton-as5916-54x/onlp/builds/src/module/src/fani.c [hunk 2]
```diff
@@ -0,0 +1,354 @@
...
+    if (onlp_file_write_int(p, path) < 0) {
+        AIM_LOG_ERROR("Unable to write data to file (%s)\r\n", path);
+        return ONLP_STATUS_E_INTERNAL;
+    }
+
+	return ONLP_STATUS_OK;
+}
+
+
+/*
```
Comment (jnealtowns) [inline packages/platforms/accton/x86-64/x86-64-accton-as5916-54x/onlp/builds/src/module/src/fani.c]: Please remove these three unsupported functions from the source code.

File: packages/platforms/accton/x86-64/x86-64-accton-as5916-54x/onlp/builds/src/module/src/ledi.c [hunk 3]
```diff
@@ -0,0 +1,256 @@
...
+    }
+
+    return ONLP_STATUS_OK;
+}
+
+/*
+ * Generic LED ioctl interface.
+ */
+int
+onlp_ledi_ioctl(onlp_oid_t id, va_list vargs)
```
Comment (jnealtowns) [inline packages/platforms/accton/x86-64/x86-64-accton-as5916-54x/onlp/builds/src/module/src/ledi.c]: Remove

**Comment by jnealtowns** [comment]
@brandonchuang  The code refers to both an r0 and r1 revision for this platform. 
But there only appears to be a r1 platform-config package. Are there two revisions?

**Comment by brandonchuang** [comment]
@jnealtowns ,
There are two revisions, r0 and R1, but r0 is no longer maintained.
I had modified the code based on your comment. Could you help review again? Thanks.

File: packages/platforms/accton/x86-64/x86-64-accton-as5916-54x/onlp/PKG.yml [hunk 4]
```diff
@@ -0,0 +1,27 @@
+variables:
```
Comment (jnealtowns) [inline packages/platforms/accton/x86-64/x86-64-accton-as5916-54x/onlp/PKG.yml]: @brandonchuang If the r0 revision of this platform will not be supported can you clean this package build to support r1 only? No need to build the package for r0 in this case.

Comment (brandonchuang) [reply packages/platforms/accton/x86-64/x86-64-accton-as5916-54x/onlp/PKG.yml]: @jnealtowns , I just removed r0 part in this file. Please help verify again. Thanks.

**Comment by jnealtowns** [comment]
Thank you @brandonchuang. 
I will fix the template such that you can call it with r1.

**Pull request merged**